_posts/2017-05-14-tiler.md

```diff
@@ -18,19 +16,19 @@ One thing we found when exploring how we might take Ordnance Survey Open Data su
 
-Tiler bundles together some powerful tools, stirching them together with Python and abstracting them away into a configuration oriented approach to generating tiles. One defines a JSON config file for a series of data and then can produce tiles using that config with Tiler.
+Tiler bundles together some powerful tools, stitching them together with Python and abstracting them away into a configuration oriented approach to generating tiles. One defines a JSON config file for a series of data and then can produce tiles using that config with Tiler.
 
 ## A Consistent Environment
 
-Tiler uses [Docker](https://docs.docker.com/get-started/); a container platform. Containers allow us to bundle up complex applications as 'images' which we can build and act as a 'lightweight, stand-alone, executable package'. This allows us to combine tools such as ogr2ogr (for geographic data format translation)and Tippecanoe (for GeoJSON to vector tile translation) and harness there functionality into a streamline pipeline that works across platforms (Windows, Mac, Linux etc).
+Tiler uses [Docker](https://docs.docker.com/get-started/); a container platform. Containers allow us to bundle up complex applications as 'images' which we can build and act as a 'lightweight, stand-alone, executable package'. This allows us to combine tools such as `ogr2ogr` (for geographic data format translation) and `tippecanoe` (for GeoJSON to vector tile translation) and harness there functionality into a streamline pipeline that works across platforms (Windows, Mac, Linux etc).
 
 ## A Worked Example
 
-Let's say we took the TQ tile from the Ordnance Survey Open Map Local Dataset and we wanted to make some Vector Tiles for the woodlands layer of the data set. 
+Let's say we took the TQ tile from the Ordnance Survey Open Map Local Dataset and we wanted to make some Vector Tiles for the woodlands layer of the data set.
 
 Our first prerequisite step would be to setup Tiler. As long as we have [docker installed](https://docs.docker.com/get-started/), we can clone the repository [Tiler repository](https://www.github.com/geovation/tiler), change to it's created directory and then do `docker build -t tiler .` inside of it. This will build our docker image.
 
-Once this is complete we can look at converting the data to WGS84 with the [OSTN02](https://www.ordnancesurvey.co.uk/business-and-government/help-and-support/navigation-technology/os-net/ostn02-ntv2-format.html) transformation file, using tilers `bng2wgs84.py` script. 
+Once this is complete we can look at converting the data to WGS84 with the [OSTN02](https://www.ordnancesurvey.co.uk/business-and-government/help-and-support/navigation-technology/os-net/ostn02-ntv2-format.html) transformation file, using tilers `bng2wgs84.py` script.
```

Comment:
'tilers' to "Tiler's" and link to `bng2wgs84.py` script.  Also, search for all instances of 'tiler' and replace with 'Tiler'.